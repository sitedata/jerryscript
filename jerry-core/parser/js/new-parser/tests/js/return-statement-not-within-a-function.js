/* This test contains return statement parser negative tests. */

return;
