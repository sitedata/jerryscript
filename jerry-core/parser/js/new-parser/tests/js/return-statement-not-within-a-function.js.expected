
--- Script parsing start ---


--- Script parsing end ---

Parse error 'Return statement must be inside a function body.' at line: 3 col: 1
